ci/install-dependencies.sh

```diff
@@ -34,7 +34,7 @@ linux-clang|linux-gcc)
 	popd
 	;;
 osx-clang|osx-gcc)
-	# brew update --quiet
+	brew update >/dev/null
```

Comment:
Why is this part of this commit?

Reply:
Actually `brew update --quiet` was throwing an error due to `--quiet` therefore, I discussed this with @dscho and changed this to `brew update >/dev/null`. So that all tests pass.

Reply:
I would hope that this is no longer necessary...